### PR TITLE
B contributor add fileA

### PR DESCRIPTION
**Pull request by israelKorea**
Folked and create a file fileA.txt
The test about git-flow using Github Remote repo.

### CODE REVIEW

File: git-master/fileA.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+B contributor working on a Docker container
+
```
Comment (HaeyoonJo) [inline git-master/fileA.txt]: test comment

**Comment by israelKorea** [comment]
what do you see

**Comment by HaeyoonJo** [comment]
I have no idea

**Comment by israelKorea** [comment]
ok, what are you waiting for? yalla

**Comment by HaeyoonJo** [comment]
ok... ㅡㅡ;

**Pull request merged**